models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -598,24 +598,34 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("microbe" %in% ic.names) {
       param[which(param[, 1] == "microbeInit"), 2] <- IC$microbe
     }
-  }
 
-  else if (length(settings$run$inputs$poolinitcond$path)>0) {
+  } else if (length(settings$run$inputs$poolinitcond$path) > 0) {
     ICs_num <- length(settings$run$inputs$poolinitcond$path)
     IC.path <- settings$run$inputs$poolinitcond$path[[1]] 
     
     IC.pools <- PEcAn.data.land::prepare_pools(IC.path, constants = list(sla = SLA))
     
-    if(!is.null(IC.pools)){
+    if (!is.null(IC.pools)) {
       IC.nc <- ncdf4::nc_open(IC.path) #for additional variables specific to SIPNET
+      ic_ncvars_wanted <- c(
+        "nee",
+        "SoilMoistFrac",
+        "SWE",
+        "date_of_budburst",
+        "date_of_senescence",
+        "Microbial Biomass C"
+      )
```

Comment:
This is a great fix.  I would like the `ic_ncvars_wanted` as the optional function input (which could be set as `NULL` as default), and if it's `NULL`, we will use the default name list as you defined here.

Reply:
Filed this for later as https://github.com/PecanProject/pecan/issues/3554